worldedit-core/src/main/java/com/sk89q/worldedit/regions/Region.java

```diff
@@ -403,7 +403,9 @@ default IChunkSet processSet(IChunk chunk, IChunkGet get, IChunkSet set) {
         if (tx >= min.x() && bx <= max.x() && tz >= min.z() && bz <= max.z()) {
             // contains some
             boolean processExtra = false;
-            for (int layer = getMinimumY() >> 4; layer <= getMaximumY() >> 4; layer++) {
+            final int max1 = Math.max(getMinimumY(), chunk.getMinY());
+            final int min1 = Math.min(getMaximumY(), chunk.getMaxY());
+            for (int layer = max1 >> 4; layer <= min1 >> 4; layer++) {
```

Comment:
I think, something like

```java
final int minLayer = Math.max(getMinimumY(), chunk.getMinY()) >> 4;
// ...
for (int layer = minLayer; //...
```
would be more readable. It seems like the variable names are mixed up either way

Reply:
Gonna rename them later
Thanks for the input
Btw the names are not mixed up